### PR TITLE
Fixes toolbar buttons in Advanced Search mode

### PR DESCRIPTION
**Pull request by AlexSCorey**
##### SUMMARY
This addresses #8237 

It also includes some refactoring of some Add/Associate toolbar buttons that open modal or wizards.
##### ISSUE TYPE
 - Bugfix Pull Request

##### COMPONENT NAME
 - UI

### CODE REVIEW

**Comment by AlexSCorey** [comment]
recheck

File: awx/ui_next/package.json [hunk 1]
```diff
@@ -54,7 +54,7 @@
   "scripts": {
     "start": "PORT=3001 HTTPS=true DANGEROUSLY_DISABLE_HOST_CHECK=true react-scripts start",
     "build": "react-scripts build",
-    "test": "TZ='UTC' react-scripts test --coverage --watchAll=false",
+    "test": "TZ='UTC' react-scripts test --watchAll=false",
```
Comment (jlmitch5) [inline awx/ui_next/package.json]: did you mean to do this?

File: awx/ui_next/src/components/PaginatedDataList/ToolbarAddButton.jsx [hunk 2]
```diff
@@ -28,7 +28,7 @@ function ToolbarAddButton({
       <DropdownItem
         key="add"
         isDisabled={isDisabled}
-        component={linkTo ? Link : Button}
+        component={linkTo ? Link : 'Button'}
```
Comment (jlmitch5) [inline awx/ui_next/src/components/PaginatedDataList/ToolbarAddButton.jsx]: EDIT: ignore previous review.  I understand now we've moved to using 'button' everywhere for this.  I believe this should be uncapatilized though

File: awx/ui_next/src/components/UserAndTeamAccessAdd/UserAndTeamAccessAdd.jsx [hunk 3]
```diff
@@ -141,16 +144,39 @@ function UserAndTeamAccessAdd({
...
-      onNext={({ id }) =>
-        setStepIdReached(stepIdReached < id ? id : stepIdReached)
-      }
-      onSave={handleWizardSave}
-    />
+    <>
+      {isKebabified ? (
+        <DropdownItem
+          key="add"
+          component="button"
```
Comment (jlmitch5) [inline awx/ui_next/src/components/UserAndTeamAccessAdd/UserAndTeamAccessAdd.jsx:151]: Again, this should be a PF {Button} I think?

File: awx/ui_next/src/screens/Inventory/shared/InventoryGroupsDeleteModal.jsx [hunk 4]
```diff
@@ -15,83 +18,151 @@ const ListItem = styled.li`
...
+    }
+  };
+  return (
+    <>
+      {isKebabified ? (
+        <DropdownItem
           key="delete"
-          isDisabled={radioOption === null}
+          isDisabled={isDisabled || isDeleteLoading}
+          component="button"
```
Comment (jlmitch5) [inline awx/ui_next/src/screens/Inventory/shared/InventoryGroupsDeleteModal.jsx:68]: Button

**Review by jlmitch5 — CHANGES_REQUESTED** [review]

File: awx/ui_next/package.json (hunk 1 above)
Comment (AlexSCorey) [reply awx/ui_next/package.json]: no I didn't mean to. I'll remove it

File: awx/ui_next/src/components/PaginatedDataList/ToolbarAddButton.jsx (hunk 2 above)
Comment (AlexSCorey) [reply awx/ui_next/src/components/PaginatedDataList/ToolbarAddButton.jsx]: I think you are right

**Review by jlmitch5 — APPROVED** [review]
Looks good!

**Comment by unlikelyzero** [comment]
@tiagodread is taking this one.
- Need to verify that the full width of the disabled / greyed out options can take up full width of the row
- Need to ensure that we're covering a kebab in at least 2 different contexts ("add/delete" and "associate/Dissociate")
- Add a stub for testing responsivity kebabs

**Review by tiagodread — APPROVED** [review]
Tests covered by 5696

**Pull request merged**